playwright/tests/slices/slicesList.spec.ts

```diff
@@ -235,3 +242,50 @@ test("I can update the screenshot", async ({ slicesListPage, slice }) => {
     slicesListPage.updateScreenshotDialog.screenshotPlaceholder,
   ).not.toBeVisible();
 });
+
+test("I will send a tracking event when I create a slice", async ({
+  slicesListPage,
+  sliceBuilderPage,
+  procedures,
+}) => {
+  let trackCallCount = 0;
+  procedures.mock("telemetry.track", ({ args }) => {
+    const [arg] = args as [{ event: string }];
+    if (arg.event === "slice:created") trackCallCount++;
+  });
+
+  await slicesListPage.goto();
+  await slicesListPage.addSliceDropdown.click();
+  await slicesListPage.addSliceDropdownCreateNewAction.click();
+
+  const sliceName = "Slice" + generateRandomId();
+  await slicesListPage.createSliceDialog.createSlice(sliceName);
+
+  await sliceBuilderPage.checkBreadcrumb(sliceName);
+  expect(trackCallCount).toBe(1);
+});
+
+test("I will not send a tracking event when I fail to create a slice", async ({
```

Comment:
this test can be merged with the one above

- create a slice normally
- check that the event is send
- mock the create call to fail
- check that the event is not send

what we gain is that the setup for both is the same, so it feels like the same path; also, the first working gives confidence that the logic of the second one works too